orc8r/cloud/go/tools/combine_swagger/spec/combine_test.go

```diff
@@ -50,16 +51,19 @@ func TestCombine(t *testing.T) {
 	cleanup()
 	defer cleanup()
 
-	cfgs, common, err := spec.Load(cfgsDir, commonFilepath)
+	specs, commonSpec, err := spec.Load(cfgsDir, commonFilepath)
+	assert.NoError(t, err)
+
+	combinedSpec, err, errs := swagger.Combine(commonSpec, specs)
 	assert.NoError(t, err)
 
-	combined, errs := spec.Combine(common, cfgs)
 	assert.Error(t, errs)
 	merrs, ok := errs.(*multierror.Error)
 	assert.True(t, ok)
 	assert.Len(t, merrs.Errors, nErrsFromCombine)
 
-	spec.Write(combined, outFilepath)
+	err = spec.Write(combinedSpec, outFilepath)
+	assert.NoError(t, err)
```

Comment:
Here/below: notice how the tool code's tests are testing I/O-wise. This is what we want. Meanwhile, the lib code is not testing I/O-wise -- it's performing the equality tests directly on the values returned from whichever relevant function